packages/web/src/components/tracks-table/TrackTableLineup.tsx

```diff
@@ -62,12 +61,11 @@ export const TrackTableLineup = ({
     loadNextPage,
     isPlaying,
     isInitialLoading,
-    data,
+    data: trackIds,
     hasNextPage,
     pageSize
   } = lineupQueryData
-
-  const tracks = data as UserTrackMetadata[] | null
+  const { data: tracks } = useTracks(trackIds?.map((entry) => entry.id) ?? [])
```

Comment:
nit: for future, `trackIds` accepts `null | undefined` so you don't need the `?? []`